code/game/gamemodes/objective.dm

```diff
@@ -287,6 +287,46 @@
 	return 1
 
 
+/datum/objective/purge
+	explanation_text = "Ensure no mutant humanoid species are present aboard the escape shuttle."
+	dangerrating = 25
+	martyr_compatible = 1
+
+/datum/objective/purge/check_completion()
+	if(SSshuttle.emergency.mode < SHUTTLE_ENDGAME)
+		return 1
+
+	var/area/A = SSshuttle.emergency.areaInstance
+
+	for(var/mob/living/player in player_list)
+		if(get_area(player) == A && player.mind && player.stat != DEAD && istype(player, /mob/living/carbon/human))
+			var/mob/living/carbon/human/H = player
+			if(H.dna.species.id != "human")

```

Comment:
It is probably fine for the aforementioned "tiny mutant parts are very hard to track or see"